### DISCUSSION TITLE
Сборка под ОС Аврора

### DISCUSSION THREAD

**Discussion by codediveRu** [anilibria/anilibria-winmaclinux#195] [Идеи]
Добрый день,
хотел бы предложить попробовать сделать сборку приложения под ОС Аврора (5.1+) - там связка QT и C++ считается нативом, насколько я понимаю. С пятой версии доступен rustore. В Аврора SDK есть эмулятор, но, если нужно, у меня есть физическое устройство (f+ r570e) - могу дополнительно и на нем потестировать.

**Comment by trueromanus** [comment]
Добрый день, да вроде никто не может Вам препятствовать :)
Можете изучить как делается сборка для других дистрибутивов вот по этой [ссылке](https://github.com/anilibria/anilibria-winmaclinux?tab=readme-ov-file#%D1%81%D0%B1%D0%BE%D1%80%D0%BA%D0%B0-%D0%B8%D0%B7-%D0%B8%D1%81%D1%85%D0%BE%D0%B4%D0%BD%D0%B8%D0%BA%D0%BE%D0%B2). По идее на авроре почти все шаги за исключение установки зависимостей должны быть полностью совместимы.
Если возникнут трудности со сборкой можете заводить ишью. Также можете приходить в группу в [телеграмм](https://t.me/+Le_oNL4Tw745YWUy) там сидят ребята обладающие разным опытом и могут попробовать коллективным разумом решить какие-то проблемы.

**Reply by codediveRu** [reply]
хорошо, спасибо за ссылки. Просто мой профиль - веб, в мобилке и qt я не разбирался, видимо пора начать :D
